worker/obuilder_build.ml

```diff
@@ -143,3 +146,13 @@ let healthcheck t =
 let cache_stats t =
   let Builder ((module Builder), builder) = t.builder in
   Builder.cache_stats builder
+
+let purge t =
+  let Builder ((module Builder), builder) = t.builder in
+  let before = Unix.gettimeofday () +. prune_margin |> Unix.gmtime in
+  (* set a future time and a big number to ensure everything is deleted *)
+  Builder.prune builder ~before Int.max_int >>= fun n ->
+  Lwt.return n
+
+let backend t =
+  t.sandbox_config
```

Comment:
iiuc, the only use of this accessor is at https://github.com/ocurrent/ocluster/pull/242/files#diff-5b7638a06391f25c3c893df03fe8754d5c0702dfc2aa4e12c7881dcf077e106eR244 to classify the kind of configuration being used, which then let's us decide whether to actually call `purge` or to return `0` in a no-op. As such, it looks to me like there is no reason to leak this out of the `Obuilder_build` API. Instead, we can just handle that logic in the `purge` function itself. WDYT?